core/src/main/java/org/elasticsearch/search/sort/NestedSortBuilder.java

```diff
@@ -47,26 +47,15 @@ public NestedSortBuilder(String path) {
     }
 
     public NestedSortBuilder(StreamInput in) throws IOException {
-        read(in);
-    }
-
-    public NestedSortBuilder(NestedSortBuilder other) {
-        if (other != null) {
-            this.path = other.path;
-            this.filter = other.filter;
-            this.nestedSort = other.nestedSort;
-        }
+        path = in.readOptionalString();
```

Comment:
If this isn't in any released version yet and ppl. agree on making `path` mandatory this could also be made non-optional.

Reply:
The path is a required option and this is unreleased code so :+1: to make field and write mandatory.